app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Show the user details by ID</h1>
```

Comment:
- As requested by the previous reviewer please use the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABWMEJKYQLW77NGOAZV7IOFIYX3WKDQ) for the user show page